contracts/LifCrowdsale.sol

```diff
@@ -265,24 +266,24 @@ contract LifCrowdsale is Ownable, Pausable {
 
   // @return true if the transaction can buy tokens on TGE
   function validPurchase() internal constant returns (bool) {
-    uint256 current = block.number;
-    bool withinPeriod = current >= startBlock && current <= endBlock2;
+    uint256 currentTime = block.timestamp;
+    bool withinPeriod = currentTime >= startTime && currentTime <= endTime2;
     bool nonZeroPurchase = msg.value != 0;
     return (withinPeriod && nonZeroPurchase);
   }
 
   // @return true if the transaction can buy tokens on presale
   function validPresalePurchase() internal constant returns (bool) {
-    uint256 current = block.number;
-    bool withinPublicPresalePeriod = current >= publicPresaleStartBlock && current <= publicPresaleEndBlock;
+    uint256 currentTime = block.timestamp;
```

Comment:
I'm not a fan of the name of this var, I'd use `now` instead or just remove it and use `block.timestamp` right away

Reply:
we can use `now` is a global var that refers to block.timestamp

Reply:
👍🏼

Reply:
👍🏼